docs/developer/releasing.md

```diff
@@ -40,8 +45,10 @@ For version v0.x.y:
    images and fat manifest with `REGISTRY="gcr.io/k8s-cluster-api" make all-push` (on the 0.1 release branch, we
    do `make docker-push`)
 7. Create an annotated tag from this same commit `git tag -a v0.x.y -m v0.x.y` and push the tag to the github repository `git push origin v0.x.y`
```

Comment:
I know this isn't new in this PR, but shouldn't we create the tag **before** building the image? I'm guessing if there's any code that sets ldflags for version based on `git describe` output, we'd need the tag there first.

Reply:
done